beacon-chain/slasher/receive_test.go

```diff
@@ -331,40 +329,7 @@ func TestSlasher_receiveAttestations_OK(t *testing.T) {
 		att1,
 		att2,
 	}
-	require.DeepEqual(t, wanted, s.attestationQueue)
-}
-
-func TestSlasher_receiveAttestations_OnlyValidAttestations(t *testing.T) {
```

Comment:
Test no longer relevant as validation no longer happens in receiveAttestations but rather in processQueuedAttestations